content/en/serverless/distributed_tracing/serverless_trace_propagation.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Serverless Trace Propagation
+kind: documentation
+further_reading:
+- link: "/tracing/"
+  tag: "Documentation"
+  text: "Explore Datadog APM"
+- link: "/tracing/trace_search_and_analytics/#live-search-for-15-minutes"
+  tag: "Documentation"
+  text: "Live Search"
+---
+
+
+{{< img src="serverless/lambda-non-http-trace.png" alt="Serverless Distributed Non-HTTP Trace"  style="width:100%;" >}}
+
+## Required Setup
+
+Additional instrumentation is sometimes required to see a single, connected trace in Node and Python serverless applications asynchronously triggering Lambda functions. If you are just getting started with monitoring serverless applications in Datadog, first [follow our main installation steps][1] and [read this page on choosing your tracing library][2]. Once you are sending traces from your Lambda functions to Datadog using the [Datadog Lambda Library][3], you may want to follow these steps to connect traces between two Lambda functions in cases such as:
+- Asynchronously triggering Lambda functions via SNS, Kinesis or EventBridge
+- Invoking Lambda functions via non-HTTP protocols such as MQTT
+
+To successfully connect trace context between resources sending traces, you need to:
+- Include Datadog trace context in outgoing events. The outgoing event can originate from a host or Lambda function with `dd-trace` installed.
```

Comment:
The sample injection code we put out only work in lambda, specifically the `get_dd_trace_context` helper function. The original code you had in the sample projects do work for both though, but they are a little bit more complicated.